src/config/navigation/tabConfig.tsx

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable react-native/no-inline-styles */
+import { upperFirst } from 'lodash';
```

Comment:
please use `import _upperFirst from 'lodash/upperFirst';` as we did already in other places